src/storage/index/index_builder.cpp

```diff
@@ -10,6 +8,7 @@
 #include "storage/index/hash_index.h"
 #include "storage/index/hash_key.h"
 #include "storage/index/index.h"
+#include "storage/index/index_builder.h"
```

Comment:
Not sure this change was intended. The current formatting is actually due to a newer clang-format which is what we'll move to at some point anyway. Regardless, I would undo this change (since it should pass current clang-format as is) so as not to increase the diff/git blame.

Reply:
Changed